toolchain/lex/lexer.cpp

```diff
@@ -952,8 +952,7 @@ auto Lexer::LexWordAsTypeLiteralToken(llvm::StringRef word, int column)
   return token;
 }
 
-[[gnu::noinline]] auto Lexer::CloseInvalidOpenGroups(TokenKind kind,
-                                                     ssize_t position) -> void {
```

Comment:
This is on an error path and so even if it's performance neutral, keeping it out-of-line makes the generated code (much) easier to understand when profiling. I'd prefer to keep it.